linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/conf/HiveEngineConfiguration.scala

```diff
@@ -40,4 +40,11 @@ object HiveEngineConfiguration {
   ).getValue
 
   val HIVE_ENGINE_TYPE = CommonVars[String]("wds.linkis.hive.engine.type", "mr").getValue
+
+  val HIVE_ENGINE_CONCURRENT_LIMIT =
+    CommonVars[Int]("linkis.hive.engineconn.concurrent.limit", 10).getValue
+
+  val HIVE_ENGINE_CONCURRENT_SUPPORT =
+    CommonVars[Boolean]("linkis.hive.engineconn.concurrent.support", true).getValue
```

Comment:
The first version is recommended to be set to false first, and then adjusted to the default concurrent execution

Reply:
Thanks for your advice, I would change the default value to false.

Reply:
> The first version is recommended to be set to false first, and then adjusted to the default concurrent execution

I have already changed the default linkis.hive.engineconn.concurrent.support value to false.

Reply:
Get